install-nimbus-cli.sh

```diff
@@ -167,6 +169,9 @@ cleanup() {
 
 success() {
   echoProgress "Success!"
+  if [[ "$IS_UPGRADE" == "true" ]] ; then
+    echoInfo "To see What's New, visit: https://experimenter.info/nimbus-cli/whats-new"
```

Comment:
This link won't be live until https://github.com/mozilla/experimenter-docs/pull/454 lands.